libs/components/molecules/src/lib/footer/footer.component.ts

```diff
@@ -0,0 +1,56 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, HostBinding, Input, Output } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { MatIconModule } from '@angular/material/icon';
+import { MatButtonModule } from '@angular/material/button';
+import { MatSelectModule } from '@angular/material/select';
+
+/** Downloading labels */
+export interface DropdownItemList {
+  label: string;
+}
+
+@Component({
+  selector: 'hra-footer',
+  standalone: true,
+  imports: [CommonModule, MatButtonModule, MatIconModule, MatSelectModule],
+  templateUrl: './footer.component.html',
+  styleUrls: ['./footer.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class FooterComponent<List extends DropdownItemList = DropdownItemList> {
+  @HostBinding('class') readonly clsName = 'hra-footer';
+
+  @Output() readonly buttonClick = new EventEmitter<void>();
```

Comment:
Inputs and outputs need improved naming to better indicate what they are used for